### PR TITLE
Remove route for changing settings and dependent code

### PR DESCRIPTION
**Pull request by adrianmroz**

### CODE REVIEW

File: src/server/config.ts [hunk 1]
```diff
@@ -260,17 +260,11 @@ if (serverSettingsFilePath) {
...
         var settingsFilePath = path.resolve(anchorPath, settingsLocation.uri);
-        if (settingsLocation.getReadOnly()) {
-          settingsStore = SettingsStore.fromReadOnlyFile(settingsFilePath, settingsLocation.getFormat());
-        } else {
-          settingsStore = SettingsStore.fromWritableFile(settingsFilePath, settingsLocation.getFormat());
-        }
+        settingsStore = SettingsStore.fromReadOnlyFile(settingsFilePath, settingsLocation.getFormat());
         break;
 
       case "mysql":
```
Comment (mkuthan) [inline src/server/config.ts]: I would remove cases other than "file" :)

**Comment by l2dy** [comment]
Merged into 1.13.0 but not in release notes?

**Comment by adrianmroz** [comment]
This is dead code elimination, you think it should be in release notes?

**Comment by l2dy** [comment]
Just asking because it's in the 1.14 milestone :)

**Pull request merged**